aws/resource_aws_api_gateway_method.go

```diff
@@ -226,6 +236,22 @@ func resourceAwsApiGatewayMethodUpdate(d *schema.ResourceData, meta interface{})
 		})
 	}
 
+	if d.HasChange("request_validator_id") {
+		var request_validator_id *string
+		if v, ok := d.GetOk("request_validator_id"); ok {
+			// requestValidatorId cannot be an empty string; it must either be nil
+			// or it must have some value. Otherwise, updating fails.
+			if s := v.(string); len(s) > 0 {
+				request_validator_id = &s
+			}
+		}
```

Comment:
I think we need to compare the old value here with the new one. If the old one is an ID, and the new one is `""`, then we're deleting a request valuator, and we should issue a `remove` update here on the `method`, correct? It doesn't look like we can otherwise remove anything as-is